index.html

```diff
@@ -184,9 +184,14 @@ <h2 class="about-myself-title">About <br> Myself</h2>
         <details class="frame f-language-list">
             <summary class="frame-summary">Languages</summary>
             <div class="summary-list">
-                <img src="images/js-lang.svg" alt="">
-                <img src="images/html-lang.svg" alt="">
-                <img src="images/css-lang.svg" alt="">
+                <div class="language-frame">
+                    <img src="images/js.png" alt="">javaScript
+                </div>
+                <div class="language-frame">
+                    <img src="images/html.png" alt="">HTML5
+                </div> <div class="language-frame">
+                    <img src="images/css.png" alt="">CSS3
+                </div>
             </div>
         </details>
```

Comment:
- [ ] Consider following the design template in Figma as what you have here is different from Figma.

Your page:
<img width="501" alt="Loom_fD1rbipwO8" src="https://user-images.githubusercontent.com/81468675/149595931-e0e16815-502e-4a8d-a094-2627c484a244.png">

Figma:
<img width="403" alt="Loom_DEoD6Uu8JR" src="https://user-images.githubusercontent.com/81468675/149595958-8a609361-3afa-40cd-bf7b-1c59542f208e.png">

Reply:
Thanks for alerting me. I used the details tag in HTML5 to create it as a widget that can be opened and closed. By default, it is closed. When open, it expands and displays the contents within. Please kindly click on the arrow to open or close. Thanks